elastalert/elastalert.py

```diff
@@ -775,13 +776,20 @@ def init_rule(self, new_rule, new=True):
                 continue
             new_rule[prop] = rule[prop]
 
-        if self.is_five():
-            self.modify_rule_for_ES5(new_rule)
+        self.modify_rule_for_ES5(new_rule)
 
         return new_rule
 
     @staticmethod
     def modify_rule_for_ES5(new_rule):
+        # Get ES version per rule
+        rule_es = elasticsearch_client(new_rule)
```

Comment:
You could probably check for the presence of 'five' on the rule before creating the client and figuring out the version, but this might be an unnecessary micro-optimization.